addons/website/static/src/systray_items/edit_website.js

```diff
@@ -61,7 +78,7 @@ class EditWebsiteSystray extends Component {
 
 export const systrayItem = {
     Component: EditWebsiteSystray,
-    isDisplayed: (env) => env.services.website.currentWebsite.metadata.editable,
+    isDisplayed: (env) => env.services.website.currentWebsite.metadata.editable || (env.services.website.currentWebsite && env.services.website.currentWebsite.metadata.translatable),
```

Comment:
Since I combined `translate_website.js` and `edit_website.js` in a same file, I thought we had to keep it as it was :
https://github.com/odoo/odoo/pull/151356/files#diff-b038a51402b0f40d654996cea971c4aafe939f9571f858ec14ca957969685243L32
I've updated the code and it seems to work fine 👍